### PR TITLE
Copyright Updates

### PR DESCRIPTION
**Pull request by cacel**

### CODE REVIEW

File: GoogleCast/BasicCastPlayer/swift/BasicCastPlayer.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -51,11 +51,19 @@
 /* End PBXFrameworksBuildPhase section */
 
 /* Begin PBXGroup section */
+		96A56DA5CA88420DAF76B31D /* Pods */ = {
```
Comment (jblaker) [inline GoogleCast/BasicCastPlayer/swift/BasicCastPlayer.xcodeproj/project.pbxproj:54]: Oops! Let's get this removed.

Comment (jblaker) [reply GoogleCast/BasicCastPlayer/swift/BasicCastPlayer.xcodeproj/project.pbxproj:54]: Looks like there are a few instances of this, let's get them all cleaned up

Comment (timothyrodgers) [reply GoogleCast/BasicCastPlayer/swift/BasicCastPlayer.xcodeproj/project.pbxproj:54]: Nice catch. After `pod install`, there are project modifications - new file groups get added and new build phases get added.

File: IMA/NativeControlsIMAPlayer_tvOS/swift/NativeControlsIMAPlayer_tvOS.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -281,7 +291,7 @@
 			buildSettings = {
 				ASSETCATALOG_COMPILER_APPICON_NAME = "App Icon & Top Shelf Image";
 				CODE_SIGN_STYLE = Automatic;
-				DEVELOPMENT_TEAM = XNV3M98BQL;
```
Comment (jblaker) [inline IMA/NativeControlsIMAPlayer_tvOS/swift/NativeControlsIMAPlayer_tvOS.xcodeproj/project.pbxproj:284]: Thanks for fixing these!

File: IMA/NativeControlsIMAPlayer_tvOS/swift/NativeControlsIMAPlayer_tvOS.xcodeproj/project.pbxproj [hunk 3]
```diff
@@ -281,7 +291,7 @@
...
+				DEVELOPMENT_TEAM = "";
```
Comment (timothyrodgers) [inline IMA/NativeControlsIMAPlayer_tvOS/swift/NativeControlsIMAPlayer_tvOS.xcodeproj/project.pbxproj:294]: Nice find.

File: GoogleCast/BasicCastPlayer/swift/BasicCastPlayer.xcodeproj/project.pbxproj (hunk 1 above)
Comment (jblaker) [reply GoogleCast/BasicCastPlayer/swift/BasicCastPlayer.xcodeproj/project.pbxproj:54]: Running `pod deintegrate` (for each project) should take care of these easy peasy.

File: SSAI/SLS_IMA-tvOSPlayer/swift/SLS_IMA-tvOSPlayer/ViewController.swift [hunk 4]
```diff
@@ -70,7 +70,7 @@ class ViewController: UIViewController {
         let imaPlaybackSessionOptions = [kBCOVIMAOptionIMAPlaybackSessionDelegateKey: self]
         
         let basicSessionProvider = manager.createBasicSessionProvider(with: nil)
-        let imaSessionProvider = manager.createIMASessionProvider(with: imaSettings, adsRenderingSettings: nil, adsRequestPolicy: adsRequestPolicy, adContainer: playerView.contentOverlayView, companionSlots: nil, upstreamSessionProvider: basicSessionProvider)
+        let imaSessionProvider = manager.createIMASessionProvider(with: imaSettings, adsRenderingSettings: nil, adsRequestPolicy: adsRequestPolicy, adContainer: playerView.contentOverlayView, viewController: self, companionSlots: nil, upstreamSessionProvider: basicSessionProvider)
```
Comment (jblaker) [inline SSAI/SLS_IMA-tvOSPlayer/swift/SLS_IMA-tvOSPlayer/ViewController.swift:73]: Nice find 👍

**Pull request merged**